development/tools/cmd/githubrelease/main.go

```diff
@@ -0,0 +1,112 @@
+// See https://cloud.google.com/docs/authentication/.
+// Use GOOGLE_APPLICATION_CREDENTIALS environment variable to specify
+// a service account key file to authenticate to the API.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"github.com/google/go-github/github"
+	"github.com/kyma-project/test-infra/development/tools/pkg/common"
+	"github.com/kyma-project/test-infra/development/tools/pkg/file"
+	"github.com/kyma-project/test-infra/development/tools/pkg/githubrelease"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2"
+)
+
+var (
+	targetCommit           = flag.String("targetCommit", "", "Target commitish [Required]")
+	bucketName             = flag.String("bucketName", "kyma-prow-artifacts", "Google bucket name where artifacts are stored [Optional]")
+	kymaConfigCluster      = flag.String("kymaConfigCluster", "kyma-config-cluster.yaml", "Filename for cluster artifacts [Optional]")
+	kymaConfigLocal        = flag.String("kymaConfigLocal", "kyma-config-local.yaml", "Filename for local artifacts [Optional]")
+	kymaChangelog          = flag.String("kymaChangelog", "release-changelog.md", "Filename for release changelog [Optional]")
+	githubRepoOwner        = flag.String("githubRepoOwner", "", "Github repository owner [Required]")
+	githubRepoName         = flag.String("githubRepoName", "", "Github repository name [Required]")
+	githubAccessToken      = flag.String("githubAccessToken", "", "Github access token [Required]")
+	releaseVersionFilePath = flag.String("releaseVersionFilePath", "", "Full path to a file containing release version [Required]")
```

Comment:
to discuss: can we have parameter that contains directly release version, not path to file that contains it?